plugins/wasm-go/extensions/ai-image-reader/README.md

```diff
@@ -0,0 +1,96 @@
+---
+title: AI IMAGE READER
+keywords: [ AI网关, AI IMAGE READER ]
+description: AI IMAGE READER 插件配置参考
+
+---
+
+## 功能说明
+
+通过对接OCR服务实现AI-IMAGE-READER，目前支持Qwen模型提供OCR服务，流程如图所示：
+
+<img src=".\ai-image-reader.png"> 
+
+## 运行属性
+
+插件执行阶段：`默认阶段`
+插件执行优先级：`400`
+
+
+## 配置说明
+
+| 名称                 | 数据类型 | 填写要求 | 默认值 | 描述                                   |
+| -------------------- | -------- | -------- | ------ | -------------------------------------- |
+| `apiKey`             | string   | 必填     | -      | 用于在访问OCR服务时进行认证的令牌。    |
+| `serviceName`        | string   | 必填     | -      | 后端OCR服务名                          |
+| `servicePort`        | int      | 必填     | -      | 后端OCR服务端口                        |
+| `provider`           | string   | 必填     | -      | 后端OCR服务提供商（例如qwen）          |
```

Comment:
The current implementation provider should be Alibaba Cloud's dashscope. Calling it qwen might cause confusion with the open-source model.
Additionally, please provide an English version of the README_EN.md document.